features/support/after-hook.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const sdapi = require('./sdapi');
+
+/**
+ * After hooks
+ * @return
+ */
+function afterHooks() {
+    this.After('@apitoken', () => {
```

Comment:
Sorry, I may have missed the initial conversation.

If you need to clean-up, you'll have to do it in the `Before`. The `After` is not guaranteed to execute if a scenario fails. This is why [some of the other features have](https://github.com/screwdriver-cd/screwdriver/blob/7948251dea91758236d95a282a3a008fa6b930eb/features/step_definitions/authorization.js#L10) a `Before` defined in them and associated via tag.